package.json

```diff
@@ -87,14 +87,15 @@
     "danger": "^11.3.1",
     "esbuild": "^0.21.3",
     "eslint": "^8.57.0",
+    "@typescript-eslint/eslint-plugin": "^6.21.0",
     "husky": "^9.0.11",
```

Comment:
for this new version this needs to be in this top place for some reason

Reply:
What do you mean by in this top place?

Reply:
I mean in the top package.json